src/pages/workspace/accounting/netsuite/export/NetSuiteExportExpensesPayableAccountSelectPage.tsx

```diff
@@ -0,0 +1,93 @@
+import {useRoute} from '@react-navigation/native';
+import React, {useCallback, useMemo} from 'react';
+import type {ValueOf} from 'type-fest';
+import BlockingView from '@components/BlockingViews/BlockingView';
+import * as Illustrations from '@components/Icon/Illustrations';
+import RadioListItem from '@components/SelectionList/RadioListItem';
+import type {SelectorType} from '@components/SelectionScreen';
+import SelectionScreen from '@components/SelectionScreen';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as Connections from '@libs/actions/connections/NetSuiteCommands';
+import Navigation from '@libs/Navigation/Navigation';
+import {getNetSuitePayableAccountOptions} from '@libs/PolicyUtils';
+import type {WithPolicyConnectionsProps} from '@pages/workspace/withPolicyConnections';
+import withPolicyConnections from '@pages/workspace/withPolicyConnections';
+import variables from '@styles/variables';
+import CONST from '@src/CONST';
+import ROUTES from '@src/ROUTES';
+
+type RouteParams = {
+    expenseType: ValueOf<typeof CONST.NETSUITE_EXPENSE_TYPE>;
+};
+
+function NetSuiteExportExpensesPayableAccountSelectPage({policy}: WithPolicyConnectionsProps) {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+
+    const policyID = policy?.id ?? '-1';
+
+    const route = useRoute();
+    const params = route.params as RouteParams;
+    const isReimbursable = params.expenseType === CONST.NETSUITE_EXPENSE_TYPE.REIMBURSABLE;
+
+    const config = policy?.connections?.netsuite.options.config;
+    const currentValue = isReimbursable ? config?.reimbursablePayableAccount : config?.payableAcct;
+    const netsuitePayableAccountOptions = useMemo<SelectorType[]>(() => getNetSuitePayableAccountOptions(policy ?? undefined, currentValue), [currentValue, policy]);
+
+    const initiallyFocusedOptionKey = useMemo(() => netsuitePayableAccountOptions?.find((mode) => mode.isSelected)?.keyForList, [netsuitePayableAccountOptions]);
+
+    const updatePayableAccount = useCallback(
+        ({value}: SelectorType) => {
+            if (currentValue !== value) {
+                if (isReimbursable) {
+                    Connections.updateNetSuiteReimbursablePayableAccount(policyID, value, currentValue ?? '');
+                } else {
+                    Connections.updateNetSuitePayableAcct(policyID, value, currentValue ?? '');
+                }
+            }
+            Navigation.goBack(ROUTES.POLICY_ACCOUNTING_NETSUITE_EXPORT_EXPENSES.getRoute(policyID, params.expenseType));
+        },
+        [currentValue, policyID, params.expenseType, isReimbursable],
+    );
+
+    const listEmptyContent = useMemo(
```

Comment:
I think we should move this as a separate component considering the amount of reuse we see across all pages.